flycheck.el

```diff
@@ -1805,7 +1805,7 @@ nil otherwise."
     (if shall-enable
         (cl-pushnew checker flycheck-enabled-checkers)
       (cl-pushnew checker flycheck-disabled-checkers))
-      shall-enable))
+    shall-enable))
```

Comment:
Oh dear, my bad: d470857af86e6927a054ecab52e7bc5f695db39c 😞 

I've fixed this formatting flaw on master, so please rebase onto `master`.  Thank you so much for making me aware!